explanations/function_representation.ipynb

```diff
@@ -542,7 +542,7 @@
     "\n",
     "wealth_grid_concatenated = jnp.concatenate([wealth_grid, wealth_points_new])\n",
     "\n",
-    "vf_via_func = value_function(wealth=wealth_grid_concatenated, vf_arr=vf_arr)"
+    "vf_via_func = value_function(wealth=wealth_grid_concatenated, next_V_arr=V_arr)"
```

Comment:
`V_via_func` ?

Maybe just search the codebase for `vf_` and adjust as you see fit.